src/main/java/uk/gov/ons/census/casesvc/service/AddressTypeChangeService.java

```diff
@@ -132,4 +186,26 @@ private String deriveAddressLevel(String oldCaseType, String newCaseType) {
       throw new RuntimeException("Invalid Case Type change");
     }
   }
+
+  private void validateModification(AddressTypeChangeAddress address) {
```

Comment:
This method looks like a carbon copy of `AddressModificationService.validate()`.

Given that it's only checking two values, would it be possible to turn it into a helper method, to avoid code duplication? Not really worth me blocking the PR for though